src/signals/settings/users/Detail/components/UserForm/index.js

```diff
@@ -211,7 +203,15 @@ UserForm.propTypes = {
       note: PropTypes.string,
     }),
   }),
+  /**
+   * Callback handler called whenever form is canceled
+   * @param {FormData} formData
+   */
   onCancel: PropTypes.func,
+  /**
+   * Callback handler called whenever form is submitted
+   * @param {FormData} formData
```

Comment:
The parameter is of type `Object`, actually.
I was thinking more along the lines of this:
```jsx
/**
 * Option select callback
 * @param {Object} formData 
 * @param {Object} formData.form - Description goes here...
 * @param {Object} formData.patchData - Description goes here...
 */
```